src/main/kotlin/calculator/domain/InputTokenizer.kt

```diff
@@ -1,24 +1,10 @@
 package calculator.domain
 
-class InputTokenizer(inputString: String?) {
-    private lateinit var delimiter: String
-    private lateinit var tokens: List<String>
-
+class InputTokenizer(inputString: String) {
+    private var delimiter: String
+    private var tokens: List<String>
 
```

Comment:
```suggestion
    private val delimiter: String
    private val tokens: List<String>

```

가변인 var보다는 val로 선언하는 게 더 안전하지 않을까하네요~